src/components/Autosuggest/Autosuggest.stories.tsx

```diff
@@ -10,7 +10,7 @@ import TextInput from "../TextInput/TextInput";
  * Wrapper component just to give the Autosuggest examples more space for the
  * suggestions dropdown to display.
  */
-const StoryWrapper = ({ children }) => (
+const StoryWrapper = ({ children }: { children: JSX.Element }) => (
```

Comment:
I've seen other places where you've created an interface for a single prop - is there a reason you typed this inline here?

Reply:
I prefer to write an interface but i want to do away with the `Autosuggest` component, so maybe I decided to do it quickly rather spend too much time on something that might go away.